client/my-sites/earn/home.tsx

```diff
@@ -495,28 +508,39 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 		<Fragment>
 			{ ! hasWordAds && <QueryWordadsStatus siteId={ siteId } /> }
 			{ ! isFreePlan && <QueryMembershipsSettings siteId={ siteId } /> }
-			<PromoSection { ...promos } />
+			{ isLoading && (
+				<div className="earn__placeholder-promo-card">
+					<PromoSection
+						header={ getHeaderCard() }
+						promos={ [ getPlaceholderPromoCard(), getPlaceholderPromoCard() ] }
+					/>
+				</div>
+			) }
+			{ ! isLoading && <PromoSection { ...promos } /> }
 		</Fragment>
 	);
 };
 
 export default connect< ConnectedProps, {}, {} >(
 	( state ) => {
+		const siteId = getSelectedSiteId( state ) ?? -1;
```

Comment:
Is there any reason for the -1 default? Normally selectors handle this. When not populated this will be null, and should evaluate to false on falsey checks.

Reply:
Typescript was complaining about the possibility of a null value being passed into the `getSitePlanSlug` selector, which was why I added the fallback. The TS error doesn't seem to block the commit, though, so if we don't consider that a problem I can remove the default value.

Reply:
Oh right, totally forgot we were in a TS file here. Maybe add a quick comment?

Reply:
Oh wait to keep it falsey, we probably need this to be 0. I'll defer to you if you'd prefer the TS error, or changing the default.

Reply:
Updating to 0 works! Pushed an update for that with a comment.